pages/cheggblog.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="/stylesheets/style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About Me</title>
+</head>
+<body>
+    <header></header>
+    <nav>
+      <h1>Emily Saeli</h1>
+      <a href="index.html">Home</a>
+      <a href="about.html">About Me</a>
+      <a href="portfolio.html">Code Portfolio</a>
+      <a href="debate.html">Debate</a>
+      <a href="publications.html">Publications</a>
+      <a href="workexperience.html">Resume</a>
+    </nav>
+<p>
+The original article can’t be found on Chegg’s website anymore but others I have written can (and should) be found below: </p>
```

Comment:
Have you tried poking around with the [Wayback Machine](https://web.archive.org/) to see if they have a copy?